Functions/Gherkin.ps1

```diff
@@ -434,8 +445,48 @@ function Import-GherkinFeature {
                 }
                 & $SafeCommands["New-Object"] Gherkin.Ast.Scenario $ExampleSet.Tags, $Scenario.Location, $Scenario.Keyword.Trim(), $ScenarioName, $Scenario.Description, $Steps | Convert-Tags $Scenario.Tags
             }
-        } else {
+        } elseif ($HideStepData) {
             $Scenario
+        } else {
+            $Steps = foreach ($Step in $Scenario.Steps) {
+                [string]$StepText = $Step.Text
+                if ($Step.Argument -is [Gherkin.Ast.DocString]) {
+                    $StepText += "$([Environment]::NewLine)  `"`"`"$([Environment]::NewLine) $(foreach ($str in ($Step.Argument.Content -split [Environment]::NewLine)) { " $str$([Environment]::NewLine)" })  `"`"`""
+                }
+                if ($Step.Argument -is [Gherkin.Ast.DataTable]) {
```

Comment:
One possible improvement that could be made is to extract this functionality out into a private Function. The same could be said for where this function processes Scenario Outlines to create new Scenarios for each table row of data.